svc/pkg/mm/worker/src/workers/lobby_ready_set.rs

```diff
@@ -1,6 +1,15 @@
 use chirp_worker::prelude::*;
 use proto::backend::pkg::*;
 use serde_json::json;
+use tokio::time::Duration;
+
+/// HACK: Give the Traefik load balancer time to complete before considering the lobby ready.
+///
+/// Traefik updates every 500 ms and we give an extra 500 ms for grace.
+///
+/// See also svc/pkg/job-run/worker/src/workers/create/mod.rs TRAEFIK_GRACE_MS
```

Comment:
Nit: it is not called TRAEFIK_GRACE_MS in create/mod.rs. Also, this comment doesn't really explain the 750 and 100 values

Reply:
750 & 100 was a typo, yikes. that's why we do reviews haha